rancher-config/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '2'
+
+services:
+    image: nhsuk/pa11y-dashboard:${DOCKER_IMAGE_TAG}
+    database:
+        image: mongo:3.3
+    dashboard:
+        build:
+            context: .
+        ports:
+            - 3333:3333
+            - 3000:3000
+        environment:
+            WEBSERVICE_DATABASE: mongodb:${MONGODB_URL}
```

Comment:
Is `MONGODB_URL` in vault? It'll need to be for a value to be set and used.

Reply:
Yeah